Gradata/docs/specs/cloud-sync-and-pricing.md

```
@@ -0,0 +1,325 @@
+POST   /api/v1/events/backfill/init # start chunked backfill session
+POST   /api/v1/events/backfill/chunk
+POST   /api/v1/events/backfill/finalize
+GET    /api/v1/brain/snapshot       # materialized system.db view (Personal+)
+GET    /api/v1/brain/materialize    # server-side re-materialize trigger
+```
+
+All require `Authorization: Bearer gk_live_<key>`. All enforce HTTPS via existing `require_https`.
+
+### 3.3 Push (incremental)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)